DirectProgramming/DPC++/DenseLinearAlgebra/matrix_mul/Makefile.win

```diff
@@ -1,20 +1,20 @@
 DPCPP_CXX = dpcpp
```

Comment:
@KanclerzPiotr Do we want to rename DPCPP to SYCL in the Makefile or maybe get rid of it altogether?